packages/runtime/src/interpreter/opcode-list.ts

```diff
@@ -1288,6 +1318,8 @@ export class Txna extends Op {
 
 // push value of a field to the stack from a transaction in the current transaction group
 // push to stack [...stack, value of field]
+// NOTE: for arg="Accounts" index 0 means sender's address, and index 1 means first address from accounts
```

Comment:
add comments for application args also

Reply:
For `ApplicationArgs` index 0 means 1st application arg from array. This is the normal case. Should i add a comment for this?

Reply:
yes, i think that will be good, because we are taking index both in application and accounts. but application args index is different

Reply:
ok, fair point.